### PR TITLE
route to /settings/plan from members tab upsell

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner 

this was all @cvializ

### CODE REVIEW

**Comment by andrewconner** [comment]
Cool, looks good

File: kifi-backend/modules/shoebox/angular/src/common/services/mixpanelService.js [hunk 1]
```diff
@@ -89,6 +89,7 @@
   }
 
   function trackEvent(action, props) {
+    //$log.log(action, props);
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/services/mixpanelService.js:92]: This dude can probably be left out, unless it's common to need logs for it.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/services/mixpanelService.js:92]: (It's minified away in prod, but commented code is unmaintained code)

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/common/services/mixpanelService.js:92]: I always add it in to make sure my tracking objects are valid, so this is intentional

**Pull request merged**